### DISCUSSION TITLE
Achievements Lost

### DISCUSSION THREAD

**Discussion by sunil-9** [community/community#49058] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I had 7 achievements in my github profile now I can only see "Star Struck" and other are gone. How can i get them back?

**Comment by sabinbaniya** [comment]
Try checking if you have hidden your achievements :)

**Comment by Anmol-Baranwal** [comment, marked as answer]
You have hidden your achievements, kindly make it public as shown here: (under settings of your profile)
![image](https://user-images.githubusercontent.com/74038190/222655689-20ee2081-e33d-4526-98fe-cb41f5d1d576.png)

Then you can go here to check: https://github.com/sunil-9?tab=achievements
